holoviews/plotting/bokeh/renderer.py

```diff
@@ -155,18 +155,23 @@ def show_callback():
         return server
 
 
-    def server_doc(self, plot, doc=None):
+    @bothmethod
+    def server_doc(self_or_cls, plot, doc=None):
```

Comment:
Isn't it really ``plot_or_obj``? I would consider renaming ``plot`` to ``obj`` as a plot is a type of object...